jackson/src/main/scala/com/lightbend/lagom/internal/jackson/JacksonObjectMapperProvider.scala

```diff
@@ -47,17 +43,11 @@ class JacksonObjectMapperProvider(
   log:           Option[LoggingAdapter]
 ) extends Extension {
 
-  // ParameterNamesModule can't be used (and is not needed) for the Immutables classes
-  private val (objectMapper: ObjectMapper, objectMapperWithParameterNames: ObjectMapper) = {
-    val mapper1 = new ObjectMapper
-    val mapper2 = new ObjectMapper
+  val objectMapper: ObjectMapper = new ObjectMapper {
```

Comment:
I'm not sure about this initialisation syntax. On the one hand, it's a little cleaner than having to re-declare the val inside the block and return it at the end (as it did previously). On the other hand, I guess this syntax creates an anonymous subclass? I read somewhere that it's idiomatic Scala to do it this way, though. What do you think?